packages/appsync-emulator-serverless/README.md

```diff
@@ -8,7 +8,7 @@ It is possible to use this emulator without serverless by mirroring the structur
 
 We aim to support the majority of appsync features (as we use all of them except elastic search).
 
- - Lambda source (only tested with serverless functions)
+ - Lambda source (only tested with serverless functions, including Python)
```

Comment:
💯 

Should we say including Node, Python?

Reply:
Yes, it should... may i try to one day give support for go :P eheh